src/components/Cards/ImageCard/ImageCard.module.scss

```diff
@@ -7,20 +7,17 @@ $placeholder-background: #000;
 
 .Container {
 	--image-card-padding: 100%;
-
 	position: relative !important;
-
 	background: #{colors.$background-primary};
 	border: 1px solid #{colors.$mine-shaft};
 	border-radius: calc(#{borders.$radius-primary} / 2) !important;
-
 	cursor: pointer;
-	overflow: hidden;
+	overflow: visible;
```

Comment:
This `overflow` is causing the design feedback I gave (image canvas escaping the border).

Reply:
Good spotting. Have made change to address this in latest commit.